coderd/prebuilds/noop.go

```diff
@@ -10,41 +10,25 @@ import (
 
 type NoopReconciler struct{}
 
-func NewNoopReconciler() *NoopReconciler {
-	return &NoopReconciler{}
-}
-
-func (NoopReconciler) RunLoop(context.Context) {}
-
-func (NoopReconciler) Stop(context.Context, error) {}
-
-func (NoopReconciler) ReconcileAll(context.Context) error {
-	return nil
-}
-
+func (NoopReconciler) RunLoop(context.Context)            {}
+func (NoopReconciler) Stop(context.Context, error)        {}
+func (NoopReconciler) ReconcileAll(context.Context) error { return nil }
 func (NoopReconciler) SnapshotState(context.Context, database.Store) (*GlobalSnapshot, error) {
 	return &GlobalSnapshot{}, nil
 }
-
-func (NoopReconciler) ReconcilePreset(context.Context, PresetSnapshot) error {
-	return nil
-}
-
+func (NoopReconciler) ReconcilePreset(context.Context, PresetSnapshot) error { return nil }
 func (NoopReconciler) CalculateActions(context.Context, PresetSnapshot) (*ReconciliationActions, error) {
 	return &ReconciliationActions{}, nil
 }
 
-var _ ReconciliationOrchestrator = NoopReconciler{}
+var DefaultReconciler ReconciliationOrchestrator = NoopReconciler{}
 
-type AGPLPrebuildClaimer struct{}
+type NoopClaimer struct{}
 
-func (c AGPLPrebuildClaimer) Claim(context.Context, database.Store, uuid.UUID, string, uuid.UUID) (*uuid.UUID, error) {
+func (NoopClaimer) Claim(context.Context, database.Store, uuid.UUID, string, uuid.UUID) (*uuid.UUID, error) {
 	// Not entitled to claim prebuilds in AGPL version.
 	return nil, ErrNoClaimablePrebuiltWorkspaces
 }
```

Comment:
nit: Should this be a separate error entirely?

Reply:
Are you referring to `ErrNoClaimablePrebuiltWorkspaces`? That's used elsewhere in the base PR:
https://github.com/coder/coder/pull/17458/files#diff-46f90baab52ea3ad914acbde30d656dbc8e46f5918d19bc056c445a1dc502482R654